apidoc/Titanium/UI/Window.yml

```diff
@@ -784,6 +784,9 @@ properties:
     type: Boolean
     since: 3.6.0
     availability: creation
+    deprecated:
+        since: 6.2.0
+        notes: Deprecated in AppCompat theme. The same behaviour can be achived by using Toolbar.
```

Comment:
Yes, it should work. On Titanium SDK less than 4.0.0. Material Theme was added in that version and it does not support this property.